modules/soe_paragraph_cta_list/config/install/core.entity_view_display.paragraph.cta_list.default.yml

```diff
@@ -0,0 +1,75 @@
+uuid: 7196d7f2-398a-4822-9eca-1ecd9395a26b
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.paragraph.cta_list.su_cta_list_deck
+    - field.field.paragraph.cta_list.su_cta_list_header
+    - field.field.paragraph.cta_list.su_cta_list_links
+    - paragraphs.paragraphs_type.cta_list
+  module:
+    - ds
+    - field_formatter_class
+    - link
+    - text
+third_party_settings:
+  ds:
+    layout:
+      id: pattern_cta-list-paragraph
+      library: null
+      disable_css: false
+      entity_classes: no_classes
+      settings:
+        pattern:
+          field_templates: default
+          variant: default
+    regions:
+      header:
+        - su_cta_list_header
+      deck:
+        - su_cta_list_deck
+      links:
+        - su_cta_list_links
+_core:
+  default_config_hash: PLmkh9jzI6OU7Zi057Qf3WyUwD-vIVaEe0MnzqBfDms
```

Comment:
These statements should be removed from the module.

Reply:
Removed.